### PR TITLE
Menu flex update

### PR DESCRIPTION
**Pull request by artjynx**
updated the justification in the mega menus. adjusted spacing

### CODE REVIEW

File: src/app/shared/styles/tl-variables.scss [hunk 1]
```diff
@@ -50,7 +50,7 @@
 
   --wvr-font-family-sans-serif: var(--tl-font-family-sans-serif);
 
-  font-family: var(--tl-font-family-sans-serif);
+  font-family: var(--tl-default-font-family-sans-serif);
```
Comment (jeremythuff) [inline src/app/shared/styles/tl-variables.scss:53]: Can this be reverted to `--tl-font-family-sans-serif`

File: src/app/tl-mega-menu/tl-mega-menu-section/tl-mega-menu-section.component.scss [hunk 2]
```diff
@@ -19,14 +20,22 @@
...
+         
         a {
-          padding: 4px 0px 4px 25px;
+          padding: 4px 0px 6px 25px;
+          margin-bottom:2px;
           color: var(--tl-deep-grey);
           font-weight: lighter;
           width: 100%;
-          display: block;
+          display: block;
```
Comment (jeremythuff) [inline src/app/tl-mega-menu/tl-mega-menu-section/tl-mega-menu-section.component.scss:38]: Please remove the extra spaces at the end of this line.

File: src/app/tl-mega-menu/tl-mega-menu-section/tl-mega-menu-section.component.scss [hunk 3]
```diff
@@ -37,5 +46,4 @@
       }
     }
   }
-}
-
+}
```
Comment (jeremythuff) [inline src/app/tl-mega-menu/tl-mega-menu-section/tl-mega-menu-section.component.scss:49]: Out linter complains if there is not a new line at the end of each file.

**Review by jeremythuff — CHANGES_REQUESTED** [review]

File: src/app/tl-mega-menu/tl-mega-menu-section/tl-mega-menu-section.component.scss (hunk 2 above)
Comment (artjynx) [reply src/app/tl-mega-menu/tl-mega-menu-section/tl-mega-menu-section.component.scss]: it looks like this is outdated - i went to the files and these had already been corrected

File: src/app/tl-mega-menu/tl-mega-menu-section/tl-mega-menu-section.component.scss (hunk 3 above)
Comment (artjynx) [reply src/app/tl-mega-menu/tl-mega-menu-section/tl-mega-menu-section.component.scss:49]: same as the other

File: src/app/shared/styles/tl-variables.scss (hunk 1 above)
Comment (artjynx) [reply src/app/shared/styles/tl-variables.scss]: this had already been updated - i'll redo the pull request - there is an outdated label on there

**Review by jeremythuff — APPROVED** [review]

**Pull request merged**